sys/net/routing/rpl/Makefile

```diff
@@ -1,8 +1,16 @@
-MODE = $(shell grep "RPL_DEFAULT_MOP RPL_NON_STORING_MODE" ../../include/rpl/rpl_config.h)
-ifneq (,$(MODE))
-	DIRS += rpl_nonstoring
+MODE = $(shell grep "\#define $(RPL_MOP)" ../../include/rpl/rpl_config.h | sed 's/\#define \s*//g')
```

Comment:
hmm, but how does the user knows/remembers which number represents which mode.
I find it easier to remember that `RPL_MOP_STORING_MODE_NO_MC` stands for storing mode wo multicast than that `0x02` does